pages/dashboard.tsx

```diff
@@ -0,0 +1,27 @@
+import NumberDisplay from '../components/NumberDisplay';
+import Warnings from '../components/Warnings';
+import Graph from '../components/Graph';
+import React from 'react';
+
+const Dashboard = () => {
+  return (
+    <div className="bg-black w-screen h-screen px-4">
+      <div className="mt-24 text-ocf-gray text-3xl font-bold">Dashboard</div>
```

Comment:
While not a styling concern, given this text here is the main page title, making it an `h1` makes this code easier to skim review wise and allows screen readers to identify it as the page title too.

```suggestion
      <h1 className="mt-24 text-ocf-gray text-3xl font-bold">Dashboard</h1>
```